ZupIT/ritchie-cli
# Adds question about metrics in init

**- What I did**
Created the metrics contract in the init command flow.

> ![metrics](https://user-images.githubusercontent.com/14079465/89951540-00d18d80-dc02-11ea-9b0b-33cae288d46e.gif)


**- How to verify it**
To check the functionality it is indicated:
1. Clone the ritchie-cli repository:
 `git clone https://github.com/ZupIT/ritchie-cli`
2. Clone this PR with the command:
 `git fetch origin pull/399/head:feature/add-question-about-sending-metrics-in-init
3. Execute `rit init`


**- Description for the changelog**

Created the metrics contract in the init command flow.
